test/unit/model_test.rb

```diff
@@ -982,12 +982,9 @@ def without(&block)
   if ::ActiveRecord::VERSION::MAJOR < 5
```

Comment:
Kept this around so you could see I hadn't changed much in the tests. I'll amend it after a review.